spec.bs

```diff
@@ -2909,6 +2913,10 @@ content in the <{fencedframe}> or its embedder, exactly one of two things will h
     1. Set |browsingContext|'s [=browsing context/fenced frame config instance=] to
        |navigationParams|'s [=navigation params/fenced frame config instance=].
 
+    1. Set |permissionsPolicy| to the result of running [=substitute the src-origin allowlist in
+       permissions declarations=] given |permissionsPolicy| and |navigationParams|'s [=navigation
+       params/fenced frame config instance=]'s [=fenced frame config/mapped url=]'s [=url/origin=].
+
   Add a new step after step 9 that reads:
```

Comment:
Yes. The adjustments above were done because we now need the permissions policy to be in place before the new step (since it's now modifying the existing permissions policy). The permissions policy is created in [step 3](https://html.spec.whatwg.org/multipage/document-lifecycle.html#initialise-the-document-object:~:text=Let%20permissionsPolicy%20be%20the%20result%20of%20creating%20a%20permissions%20policy%20from%20a%20response%20given%20navigationParams%27s%20navigable%27s%20container%2C%20navigationParams%27s%20origin%2C%20and%20navigationParams%27s%20response.%20%5BPERMISSIONSPOLICY%5D), so the new step has to be step 4.